### PR TITLE
UI: Improved Add tag UX and added services search-bar back Also fixed services loader issue

### LINKED ISSUE #852: Dataset entity page , Schema tab, Tags column doesn't indicate there is a way to add a tag unless user hovers over it
image
we need to show some indication that there is a way to add tags. By just looking at that page it looks like a read-only page then any actions can be taken unless the user hovers over the tags column or description column
cc @darth-coder00 @sureshms

### PR DESCRIPTION
**Pull request by darth-coder00**
### Describe your changes :
Closes #852 

I worked on the Schema tab and services page because need to fix UX for `+ Add tag`, services search-bar, services page loading flicker

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
Screenshot 2021-11-03 at 6 39 44 PM
Screenshot 2021-11-03 at 6 40 10 PM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t @Sachin-chaurasiya

### CODE REVIEW

File: catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx [hunk 1]
```diff
@@ -516,9 +516,9 @@ const EntityTable = ({
                                     />
                                   </button>
                                 ) : (
-                                  <span className="tw-opacity-0 group-hover:tw-opacity-100">
+                                  <span className="tw-opacity-60 group-hover:tw-opacity-100 tw-text-grey-muted group-hover:tw-text-primary">
                                     <Tags
-                                      className="tw-border-main"
+                                      className=""
```
Comment (Sachin-chaurasiya) [inline catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx:521]: if there is no class then we should remove this.

**Review by Sachin-chaurasiya — COMMENTED** [review]
one minor change otherwise LGTM.

File: catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx (hunk 1 above)
Comment (darth-coder00) [reply catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx:521]: My bad :)

Comment (darth-coder00) [reply catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx]: Done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**